cmd/kubeadm/app/apis/kubeadm/v1alpha3/doc.go

```diff
@@ -16,12 +16,36 @@ limitations under the License.
 
 // Package v1alpha3 is the API (config file) for driving the kubeadm binary.
 // Some of these options are also available as command line flags, but
-// the preferred way to configure kubeadm is to pass a YAML file in with the
-// --config option.
+// the preferred way to configure kubeadm is to pass a single YAML file with
+// multiple configuration types in with the --config option.
```

Comment:
Although in the future the users may have the choice to store different `kind`s in different YAML files, currently we don't. So this must be single YAML file and we should emphasize on that here.
Also, InitConfiguration and JoinConfiguration cannot share a single YAML file at the moment (kubeadm bails out). So we should probably mention that too. Not sure for ClusterConfiguration and JoinConfiguration though.

Reply:
> Not sure for ClusterConfiguration and JoinConfiguration though.

yeah, we probably want to handle all scenarios.
to exclude the possibility of related `k/kubeadm` issue reports, that is.

Reply:
i'll add some additional text to catch edge cases.